tests/end_to_end/test_vulnerability_detector/test_vulnerability_detector.py

```diff
@@ -556,7 +556,49 @@ def get_results(self):
 
     @pytest.mark.parametrize('preconditions, body, teardown', complete_list, ids=list_ids)
     def test_vulnerability_detector_scans_cases(self, setup_vulnerability_tests, request, preconditions, body, teardown, setup,
-                                                host_manager, get_results):
+                                                host_manager, get_results):        
+        """
+        description: validates the Vulnerability Detector's ability to detect new vulnerabilities in the environment for each of the defined cases.
+
+        This is the main test. It is in charge of executing each of the cases defined in test_vulnerability.yaml, 
+        checking if all agents generate the corresponding vulnerabilities and collecting the results.
+
+        tier: 1
```

Comment:
```suggestion
        tier: 0
```

Reply:
Fixed in https://github.com/wazuh/wazuh-qa/pull/5151/commits/b1da35a1fb87428a5059f7574e32fed607aa28a7